### PR TITLE
OKTA-466754 API docs update for jwks_uri

### PR DESCRIPTION
**Pull request by mymoonasaifunnissameerasahib-okta**
## Description:
- API docs update for jwks_uri 
- **Is this PR related to a Monolith release?** Yes, 2022.03.0

### Resolves:

* [OKTA-466754](https://oktainc.atlassian.net/browse/OKTA-466754)

### CODE REVIEW

**Review by larsjohansen-okta — COMMENTED** [review]
Do we want to have any mention / examples with this or just adding it as properties to App settings and client?

**Comment by mymoonasaifunnissameerasahib-okta** [comment]
> Do we want to have any mention / examples with this or just adding it as properties to App settings and client?

Updated

File: packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md [hunk 1]
```diff
@@ -177,6 +177,55 @@ Content-Type: application/json;charset=UTF-8
 }
 ```
 
+##### Request example: Create a Service App with a JWKS Uri
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oauth-clients/index.md:180]:
```suggestion
##### Request example: Create a Service App with a JWKS URI
```

**Review by susanharper-okta — COMMENTED** [review]
one nit

**Comment by mymoonasaifunnissameerasahib-okta** [comment]
> one nit

@susanharper-okta updated.

**Review by susanharper-okta — APPROVED** [review]
LGTM

**Review by larsjohansen-okta — APPROVED** [review]

**Pull request merged**